README.md

```
@@ -36,3 +36,29 @@ Run a dev build:
 - `yarn dev:core` (core package)
 - `yarn dev:transport-http` (transport-http package)
 - `yarn dev:shared` (shared package)
+
+### Development with example desktop app (Recommend)
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)